full/build.gradle

```diff
@@ -7,6 +7,14 @@ plugins {
     id 'antlr'
     id "org.sonarqube"
     id "org.jetbrains.kotlin.jvm" version "1.6.0"
+    id "com.diffplug.spotless" version "6.7.2"
+}
+
+spotless {
+    java {
+        target 'src/*/java/**/*.java'
+        removeUnusedImports()
+    }
 }
```

Comment:
This is only affecting to java files, but I believe there's some Kotlin in the full codebase

Reply:
Discussed during standup and we decided we can skip the Kotlin sources